lib/src/core/channel/guild/guild_channel.dart

```diff
@@ -50,10 +50,24 @@ abstract class IGuildChannel implements IMinimalGuildChannel {
   /// Throws if [entity] isn't [User] or [Role]
   Future<void> deleteChannelPermission(SnowflakeEntity entity, {String? auditReason});
 
-  /// Creates new [IInvite] for [IChannel] and returns it"s instance
+  /// Creates new [IInvite] for [IChannel] and returns it's instance.
+  ///
+  /// Requires the `CREATE_INSTANT_INVITE` permission.
```

Comment:
```suggestion
  /// Requires the [PermissionsConstants.createInstantInvite] permission.
```
It would be better to link directly to the permission of the lib instead of the api